nginx/nginx-proxy.conf

```diff
@@ -8,6 +8,11 @@ upstream redis-server {
     server backend:8000 fail_timeout=0;
 }
 
+upstream frontend-server {
+    ip_hash;
+    server frontend:3000 fail_timeout=0;
+}
+
```

Comment:
Just like the docker-compose file, modifications made to this file were only meant for development setup, and are not meant to be committed.